tests/js/extractors/factories/htmlTemplate.test.ts

```diff
@@ -0,0 +1,66 @@
+import { htmlTemplateExtractor } from '../../../../src/js/extractors/factories/htmlTemplate'
+import { HtmlExtractors } from '../../../../src/html/extractors';
+import { JsParser } from '../../../../src/js/parser';
+import { HtmlParser } from '../../../../src/html/parser';
+import { CatalogBuilder, IMessage } from '../../../../src/builder';
+
+describe('JS: HTML template extractor', () => {
+  describe('calling html parser ', () => {
+    let builder: CatalogBuilder;
+    let messages: IMessage[]
+    let jsParser: JsParser;
+    let htmlParser: HtmlParser;
+
+    beforeEach(() => {
+      messages = [];
+
+      builder = <any>{
+        addMessage: jest.fn((message: IMessage) => {
+          messages.push(message);
+        })
+      };
+
+      htmlParser = new HtmlParser(builder, [
+        HtmlExtractors.elementContent('translate')
+      ]);
+
+      jsParser = new JsParser(builder, [
+        htmlTemplateExtractor(htmlParser)
+      ]);
+    });
+
+    test('HTML inside a template literal', () => {
```

Comment:
I'd like to see some variety of tests. Take a look at `tests/html/extractors/factories/embeddedJs.test.ts`, you should be able to write very similar tests for this case.

Additionally, it would be good to have a test for each of the supported type of strings (regular string literal & template string literal).

Reply:
If you try to implement similar tests to the ones from embeddedJs, I expect `with lineNumberStart option` to be failing with this extractor. To resolve that, we basically have to implement this change for the js parser: #59

Reply:
I've added more tests, but I couldn't make the `separate line` one from `embeddedJs` work. I doesn't extract anything if I put `\n` inside the html

Reply:
The separate line case is already covered by the other tests you have written, so that's fine.